### ISSUE TITLE
Pixel perfect scaling options

### ISSUE THREAD

**Issue by subsoap** [rgrams/rendercam#11]
Like Enter the Gungeon for PC

Useful for pixel art games

![2018-01-11 23_19_01-enter the gungeon](https://user-images.githubusercontent.com/409170/34863762-f296288e-f725-11e7-90db-70dd258f5057.png)
![2018-01-11 23_18_35-enter the gungeon](https://user-images.githubusercontent.com/409170/34863763-f2aa5e12-f725-11e7-9db7-ff798a75783f.png)

**Comment by rgrams** [comment]
Yeah, this should definitely be supported. Have you done any experiments to see if it's possible with Rendercam as-is?

**Comment by subsoap** [comment]
Not yet.
